modules/serverless-ossm-setup.adoc

```diff
@@ -17,45 +26,187 @@ metadata:
   name: default
   namespace: istio-system
 spec:
-  members:
-    - <namespace> <1>
+  members: <1>
+    - knative-serving
+    - <namespace>
 ----
 <1> A list of namespaces to be integrated with {ProductShortName}.
-. Create a network policy that permits traffic flow from Knative system Pods to Knative services:
-.. Add the `serving.knative.openshift.io/system-namespace=true` label to the `knative-serving` namespace:
++
+[IMPORTANT]
+====
+This list of namespaces must include the `knative-serving` namespace.
+====
+. Apply the `ServiceMeshMemberRoll` resource:
 +
 [source,terminal]
 ----
-$ oc label namespace knative-serving serving.knative.openshift.io/system-namespace=true
+$ oc apply -f <filename>
 ----
-.. Add the `serving.knative.openshift.io/system-namespace=true` label to the `knative-serving-ingress` namespace:
+. Create the necessary gateways so that {ProductShortName} accepts traffic:
++
```

Comment:
Add _.Example `knative-local-gateway` using HTTP_ to distinguish from the next example.